<ui version="4.0" >
 <class>QueryEditorDialog</class>
 <widget class="QDialog" name="QueryEditorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Build Query</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Table to Query:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tableList" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>368</width>
        <height>269</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Fields</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QListView" name="columnView" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <spacer name="verticalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addAllButton" >
           <property name="text" >
            <string>>></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton" >
           <property name="text" >
            <string>></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton" >
           <property name="text" >
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeAllButton" >
           <property name="text" >
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>27</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="2" >
        <widget class="QListView" name="selectView" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>368</width>
        <height>269</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Terms</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QRadioButton" name="andButton" >
         <property name="text" >
          <string>Match all of the following terms</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="orButton" >
         <property name="text" >
          <string>Match any of the following terms</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QScrollArea" name="scrollArea" >
         <property name="widgetResizable" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>354</width>
            <height>157</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>284</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="moreButton" >
           <property name="text" >
            <string>More</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lessButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Less</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QueryEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QueryEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
